cognite/v05/timeseries.py

```diff
@@ -204,6 +204,9 @@ def _split_TimeseriesWithDatapoints_if_over_limit(timeseries_with_datapoints: Ti
     Returns:
         A list of v05.dto.TimeSeriesWithDatapoints where each has a maximum number of datapoints equal to the limit given.
     '''
+    if limit<1:
```

Comment:
Should this be allowed? The API does have an upper limit on the number of datapoints you can post at once.

Reply:
True. I mistakenly thought we had to post everything at once. But thinking it through I realised we don't as long as we know how to set the indexes. Will fix....